base/src/com/google/idea/blaze/base/buildmodifier/BuildFileFormatOnSaveHandler.java

```diff
@@ -61,7 +61,7 @@ private static void formatBuildFile(Project project, PsiFile psiFile) {
       return;
     }
     // DO not use formatter here, instead we format the entire file.
-    BlazeFileType type = ((BuildFile) psiFile).getBlazeFileType();
+    BuildFile buildFile = (BuildFile) psiFile;
```

Comment:
Can you please remove the unused import `import com.google.idea.blaze.base.lang.buildfile.psi.BuildFile.BlazeFileType;`?